docs/sources/flow/reference/components/otelcol.exporter.loadbalancing.md

```diff
@@ -0,0 +1,263 @@
+---
+title: otelcol.exporter.loadbalancing
+labels:
+  stage: beta
+---
+
+# otelcol.exporter.loadbalancing
+
+{{< docs/shared lookup="flow/stability/beta.md" source="agent" >}}
+
+`otelcol.exporter.loadbalancing` accepts logs and/or traces from other `otelcol` components
+and writes them over the network using the OpenTelemetry Protocol (OTLP) protocol. 
+
+> **NOTE**: `otelcol.exporter.loadbalancing` is a wrapper over the upstream
+> OpenTelemetry Collector `loadbalancing` exporter. Bug reports or feature requests will
+> be redirected to the upstream repository, if necessary.
+
+Multiple `otelcol.exporter.loadbalancing` components can be specified by giving them
```

Comment:
This is actually a standard line which we put in a lot of docs pages. I'm not against changing it everywhere, but for the sake of consistency I'd prefer to leave it as it is in this page for now. Maybe it can be changed for all components in a separate PR.

Reply:
Then lets leave it as-is for now - save it for a more general "cleanup" PR.